src/mgr/Action.cc

```diff
@@ -112,11 +113,17 @@ Mgr::Action::fillEntry(StoreEntry* entry, bool writeHttpHeader)
         entry->replaceHttpReply(rep);
     }
 
-    dump(entry);
+    dump(entry); // TODO: replace with report() when all children are converted
 
     entry->flush();
 
     if (atomic())
         entry->complete();
 }
 
+void
+Mgr::Action::dump(StoreEntry *entry)
+{
+    PackableStream os(*entry);
+    report(os);
```

Comment:
Besides that, why not use an overload for supporting the transition?
report() is a better name, and I'd support for it but we can do it with a different strategy:
- decide what the API will look like
- overload dump()
- progressively convert and possibly rename (or keep dump to avoid purely cosmetic changes)

Reply:
I believe C++ term "overload" is not applicable here, but the rest of your description matches my expectations of the anticipated transition. None of my blocking change requests contradict that natural transition strategy.